test/pull_request_event_payload.json

```diff
@@ -0,0 +1,13 @@
+{
+  "pull_request": {
+    "base": {
+      "sha": "22b00741ec61ff185b06449115a9a448d4db00e9"
```

Comment:
This is the sha just before Dma added in the test files with the (intentionally) bad formatting.